src/objective/rank_objective.hpp

```diff
@@ -178,43 +232,87 @@ class LambdarankNDCG : public RankingObjective {
           high_rank = j;
           low_rank = i;
         }
-        const data_size_t high = sorted_idx[high_rank];
-        const int high_label = static_cast<int>(label[high]);
-        const double high_score = score[high];
-        const double high_label_gain = label_gain_[high_label];
-        const double high_discount = DCGCalculator::GetDiscount(high_rank);
+
+        // info of more relevant doc
+        const data_size_t high = sorted_idx[high_rank];  // doc index in query results
```

Comment:
* in _old repo_ `[high/low]_rank` ...
  * ... means sorted_idx value (current score prediction) of the more relevant document 
  * in this repo those variables are called `debias_[high/low]_rank`
* in _current repo_ `[high/low]_rank` ...
  * are the loop variables of the more/less relevant document
  * this means that `debias_[high/low]_rank = sorted_idx[high_rank]` here